harness/tests/launch/test_wrap_rank.py

```diff
@@ -0,0 +1,50 @@
+import subprocess
+import sys
+import textwrap
+
+from determined.launch import wrap_rank
+
+
+def test_split_on_new_lines_or_carriage_returns() -> None:
```

Comment:
This is a little pedantic, but I like it when tests test one thing. Even if that means copy/pasted code. Here, what do you think of two tests; one for `\n` and one for `\r`?

Reply:
Possibly also one for `stdin` one for `stderr`? Maybe it's worth a short chat about what that division is testing.

Reply:
resolved; opted to remove the stderr writes.  They were intended to be signals but since the stdout reads were blocking it actually made no difference.